packages/embark-console/src/lib/index.ts

```diff
@@ -60,24 +58,11 @@ class Console {
     }
     this.events.setCommandHandler("console:executeCmd", this.executeCmd.bind(this));
     this.events.setCommandHandler("console:history", (cb: any) => this.getHistory(this.cmdHistorySize(), cb));
-    this.events.setCommandHandler("console:provider:ready", (cb: any) => {
-      if (this.providerReady || this.isEmbarkConsole) {
-        return cb();
-      }
-      this.events.once("console:provider:done", cb);
-    });
```

Comment:
This breaks running tests as it still requests for `console:provider:ready`

Judging by the code changes, I assume this even is simply obsolete now and event requests can be removed?

Reply:
Fixed with: https://github.com/embark-framework/embark/pull/1680/commits/123bb5e0ce3eafd77ba4d09239afd73d8305b61c